src/functions.js

```diff
@@ -91,6 +98,21 @@ export class TaskStorage {
             this.rearray();
           });
         });
+
+        const checkbox = li.querySelector('.checkbox');
+        checkbox.addEventListener('click', (event) => {
+          event.stopPropagation();
+          li.classList.toggle('overline');
+          console.log(li.dataset.taskId, 10);
+          const taskId = parseInt(li.dataset.taskId, 10);
+          const check = checkbox.checked;
+          const taskToUpdate = this.data.find((task) => task.index === taskId);
+          if (taskToUpdate) {
+            taskToUpdate.completed = check;
+            this.saveData();
+            this.displayData();
+          }
+        });
       });
```

Comment:
[Optional] Please kindly ensure that in future projects you use more imperative commit messages, as you have just one commit message for this your project which is not a good way of writing clean and understandable code 😄